Drivers/Src/Lights.c

```diff
@@ -66,8 +93,8 @@ void Lights_Set(light_t light, State state) {
         uint8_t currData = rxBuf[1];
```

Comment:
Shouldn't this be rxBuf[0]?

Reply:
It *might* put it in buf[1] since you passed in a length of two bytes, but I'm not sure. I would highly recommend making the rxBuf one byte long because that's all it ever needs.